.github/workflows/tests.yml

```
@@ -0,0 +1,57 @@
+name: CI
+
+on:
+  push:
+    branches: master
+  pull_request:
+    branches: master
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 3 * * 6'
+  workflow_dispatch:
+    inputs:
+      reason:
+        description: 'Reason'
+        required: false
+        default: 'Manual trigger'
+
+jobs:
+  Tests:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+          python-version: [3.6, 3.7, 3.8]
+          requirements-level: [min, pypi]
+
+    steps:
+      - name: Intall APT dependencies
+        run: sudo apt-get install libfreetype6-dev
+        
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Generate dependencies
+        run: |
+          python -m pip install --upgrade pip setuptools py wheel requirements-builder
+          requirements-builder -e ${{ matrix.EXTRAS }} ${{ matrix.REQUIREMENTS_FILE }} --level=${{ matrix.REQUIREMENTS_LEVEL }} setup.py > .${{ matrix.REQUIREMENTS_LEVEL }}-${{ matrix.python-version }}-requirements.txt
```

- We don't have `EXTRAS` because it's always `all` (at least this was the case in invenio-theme.
- `REQUIREMENTS_FILE` is not needed anymore, was only needed for `dev` and we don't have it.
- `REQUIREMENTS_LEVEL` vs `requirements-level`

```suggestion
          requirements-builder -e all  --level=${{ matrix.REQUIREMENTS_LEVEL }} setup.py > .${{ matrix.requirements-level }}-${{ matrix.python-version }}-requirements.txt
```